lib/smack/SmackInstGenerator.cpp

```diff
@@ -623,6 +623,22 @@ void SmackInstGenerator::visitCallInst(llvm::CallInst& ci) {
     Value* val = ci.getArgOperand(0);
     emit(Stmt::assign(rep->expr(&ci), rep->expr(val)));
 
+  } else if (name.find("llvm.convert.from.fp16.f32") != std::string::npos) {
```

Comment:
It seems that we can merge these branches like `if (a || b || c ...)` since they have the same body.